docs/changes/16451.other.rst

```diff
@@ -0,0 +1,3 @@
+Updated bundled WCSLIB version to 8.3. This update changes the behavior of various ``*set`` functions in order to improve stability of WCSLIB in threaded applications. For a full list of changes - see
+http://www.atnf.csiro.au/people/mcalabre/WCS/CHANGES or
```

Comment:
Let's remove the URL. I don't want to have to "change history" in the future when the URL breaks. Thanks!
```suggestion
```

Reply:
Maybe let's not re-run all the extra CI just for this change though :sweat_smile:

Reply:
@pllim https://github.com/astropy/astropy/blob/main/docs/wcs/references.txt is full of linksthat could change in the future. I think links are helpful and make life easier for those who read the docs and probably we will have to fix links when they become outdated.

Reply:
The live links in change log topic had been debated before and we decided to not do it.

Yes, no need to rerun CI, you can do a `[ci skip]` and I can merge it.